src/tasks.md

```diff
@@ -0,0 +1,263 @@
+      ```
+  
+      - Equivalent command with the policy binary downloaded: `kwctl run file://$PWD/pod-privileged-policy.wasm -r privileged-pod-request.json`
+  
+    - Output:
+  
+      ```json
+      {"uid":"8EE6AF8C-C8C8-45B0-9A86-CB52A70EC50D","allowed":false,"status":{"message":"User 'kubernetes-admin' cannot schedule privileged containers"}}
+      ```
+  
+    - Result: The policy denies the request
+  
+    > **INFO**: if you want to see a more complex example, you can read this Kubewarden blog post: [Introducing `kwctl` to Kubernetes Administrators](https://www.kubewarden.io/blog/2021/06/kwctl-intro-for-kubernetes-administrators/)
+  
+## Enforce Policies
+
+As described in the [Quick Start](/quick-start.html#example-enforce-your-first-policy), you can enforce a policy by defining a `ClusterAdmissionPolicy` and then deploy it to your cluster using `kubectl`.
+
+`kwctl` will help to generate a `ClusterAdmissionPolicy` from the policy you want to enforce. 
+
+You can follow the steps described in the [Quick Start](/quick-start.md#example-enforce-your-first-policy) after you generated the `ClusterAdmissionPolicy` and applied it to your Kubernetes cluster, as follow:
+
+  - Generate the `ClusterAdmissionPolicy` from the policy `manifest` and save it to a file
+  
+    - Command: `kwctl manifest -t ClusterAdmissionPolicy <policy URI> > <"policy name".yaml>`
+    - Example:
+    
+    ```shell
+    $ kwctl manifest -t ClusterAdmissionPolicy registry://ghcr.io/kubewarden/policies/pod-privileged:v0.1.9
+    ---
+    apiVersion: policies.kubewarden.io/v1alpha2
+    kind: ClusterAdmissionPolicy
+    metadata:
+      name: privileged-pods
+    spec:
+      module: "registry://ghcr.io/kubewarden/policies/pod-privileged:v0.1.9"
+      settings: {}
+      rules:
+        - apiGroups:
+            - ""
+          apiVersions:
+            - v1
+          resources:
+            - pods
+          operations:
+            - CREATE
+      mutating: false
+    ```
+  
+    > TIP: by default, the `name` value is set to `generated-policy`. You might want to edit it before you deploy the `ClusterAdmissionPolicy`. 
+    >
+    > NOTE: To avoid confusion, the value above has been set to `privileged-pods`.
+  
+  - Deploy the `ClusterAdmissionPolicy` to your Kubernetes cluster
+
+    - Command: `kubectl apply -f <"policy name".yaml>`
+    - Example:
+    
+    ```shell
+    $ kubectl apply -f pod-privileged-policy.yaml
+    clusteradmissionpolicy.policies.kubewarden.io/privileged-pods created
+    ```
+
+Once the `ClusterAdmissionPolicy` deployed, the requests sent to your Kubernetes cluster will be evaluated by the policy, if they're within the policy scope.
+
+## Next steps
+
+### Write Policies
+
+The section [Writing Policies](/writing-policies/) explains how to write policies in different languages and how to export them into Webassembly that can be interpreted by Kubewarden.
```

Comment:
```suggestion
The [Writing Policies](/writing-policies/) section explains how to write policies in different languages and how to export them into Webassembly so that they can be interpreted by Kubewarden.
```